.github/CONTRIBUTING.md

```diff
@@ -322,7 +322,6 @@ eg. on MacOS you can follow this instructions:
 # 2 Install nvm (https://github.com/nvm-sh/nvm#troubleshooting-on-macos) and other required dependencies
 brew update
 brew install nvm
-brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman python-setuptools

```

Comment:
no longer required, because of the node-canvas library version upgrade. they provide pre-built binaries for linux and darwin